modules/ROOT/pages/cluster-add-srv.adoc

```diff
@@ -3,21 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To add a server to a cluster, all servers in a cluster must run the same version of Mule runtime and of the Runtime Manager Agent.
+To add a server to a cluster, all servers in a cluster must run the same version of Mule runtime and of the Runtime Manager agent.
 
 [NOTE]
-Remember that you must first register each of the servers in the cluster by downloading and installing the same version of the https://www.mulesoft.com/platform/mule[Mule] and then running the xref:managing-servers.adoc#add-a-server[amc_setup] script on each.
-A cluster can only be made up of servers that are not already in a server group or part of another cluster.
+You must first register each of the servers in the cluster by downloading and installing the same version of the https://www.mulesoft.com/platform/mule[Mule] and then running the xref:managing-servers.adoc#add-a-server[amc_setup] script on each.
```

Comment:
Register the servers before what? I'd delete "You must" and just make this a step. I don't understand why this is a note when it is a requirement for something. This is a step, not a note...?